src/engine/qcommon/net_chan.cpp

```diff
@@ -59,7 +59,7 @@ to the new value before sending out any replies.
 
 */
 
-static const int MAX_PACKETLEN = 1400; // max size of a network packet
+static const int MAX_PACKETLEN = 265000; // max size of a network packet
```

Comment:
You can't just make the size of the real packet arbitrarily large. Some people suggest the current value is good for [Ethernet](https://stackoverflow.com/a/3074427). And supposedly a packet this large would be [immediately rejected](https://stackoverflow.com/a/62726938) by any of our supported operating systems. Finally [Cloudflare says](https://blog.cloudflare.com/ip-fragmentation-is-broken/) that sending any packets larger than 1500 is likely to be broken.